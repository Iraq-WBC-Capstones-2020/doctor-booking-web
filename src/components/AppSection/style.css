body {
  background-color: #F5F9FD;
}

.applicationSection {
  margin-top: 99px;
  margin-bottom: 99px;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  text-align: center;
}

.myRow {
  -webkit-box-pack: justify !important;
      -ms-flex-pack: justify !important;
          justify-content: space-between !important;
}

.applicationImage {
  margin-top: 48px;
  width: 610px;
  height: 346px;
}

.text {
  height: 381px;
  width: 320.21px;
}

.text h2 {
  color: #026FC2;
  text-align: left;
  font-family: 'Roboto', sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 36px;
  line-height: 48px;
}

.text p {
  width: 313px;
  height: 155px;
  text-align: left;
  font-family: Roboto;
  font-style: normal;
  font-weight: normal;
  font-size: 18px;
  line-height: 30px;
  color: #7D7987;
}

.downloadButtons {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  float: left;
}

.downloadButtons input {
  margin-right: 23px;
  width: 117px;
  height: 35px;
}
/*# sourceMappingURL=style.css.map */