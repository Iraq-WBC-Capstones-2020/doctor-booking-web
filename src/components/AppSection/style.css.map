{
    "version": 3,
    "mappings": "AAAA,AAAA,IAAI,CAAC;EACH,gBAAgB,EAAE,OAAO;CAC1B;;AAED,AAAA,mBAAmB,CAAC;EAClB,UAAU,EAAE,IAAI;EAChB,aAAa,EAAE,IAAI;EACnB,WAAW,EAAE,MAAM;EACnB,UAAU,EAAE,MAAM;CACnB;;AAED,AAAA,MAAM,CAAC;EACL,eAAe,EAAE,wBAAwB;CAC1C;;AAED,AAAA,iBAAiB,CAAC;EAChB,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,KAAK;CACd;;AAED,AAAA,KAAK,CAAC;EACJ,MAAM,EAAE,KAAK;EACb,KAAK,EAAE,QAAQ;CAuBhB;;AAzBD,AAIE,KAJG,CAIH,EAAE,CAAC;EACD,KAAK,EAAE,OAAO;EACd,UAAU,EAAE,IAAI;EAChB,WAAW,EAAE,oBAAoB;EACjC,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;CAClB;;AAZH,AAcE,KAdG,CAcH,CAAC,CAAC;EACA,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,KAAK;EACb,UAAU,EAAE,IAAI;EAChB,WAAW,EAAE,MAAM;EACnB,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,MAAM;EACnB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,KAAK,EAAE,OAAO;CACf;;AAGH,AAAA,gBAAgB,CAAC;EACf,OAAO,EAAE,IAAI;EACb,KAAK,EAAE,IAAI;CAOZ;;AATD,AAIE,gBAJc,CAId,KAAK,CAAC;EACJ,YAAY,EAAE,IAAI;EAClB,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,IAAI;CACb",
    "sources": [
        "style.scss"
    ],
    "names": [],
    "file": "style.css"
}