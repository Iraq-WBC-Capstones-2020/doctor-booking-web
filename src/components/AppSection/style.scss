body{
    background-color: #F5F9FD;
}

.applicationSection{
    margin-top: 99px;
    margin-bottom: 99px;
    align-items: center;
    text-align: center;
}

.myRow{
    justify-content: space-between !important;
}

.applicationImage{
    margin-top: 48px;
    width: 610px;
    height: 346px;
}

.text{
    height: 381px;
    width: 320.21px;
    
    h2{
        color: #026FC2;
        text-align: left;
        font-family: 'Roboto', sans-serif;
        font-style: normal;
        font-weight: 500;
        font-size: 36px;
        line-height: 48px;
    }

    p{
        width: 313px;
        height: 155px;
        text-align: left;
        font-family: Roboto;
        font-style: normal;
        font-weight: normal;
        font-size: 18px;
        line-height: 30px;
        color: #7D7987;
    }
}

.downloadButtons{
    display: flex;
    float: left;

    input{
    margin-right: 23px;
    width: 117px;
    height: 35px;
    }
}