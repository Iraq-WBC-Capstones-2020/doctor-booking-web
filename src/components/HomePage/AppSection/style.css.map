{
    "version": 3,
    "mappings": "AAAA,AAAA,mBAAmB,CAAC;EAClB,UAAU,EAAE,IAAI;EAChB,aAAa,EAAE,IAAI;EACnB,WAAW,EAAE,MAAM;EACnB,UAAU,EAAE,MAAM;CACnB;;AAED,AAAA,MAAM,CAAC;EACL,eAAe,EAAE,wBAAwB;CAC1C;;AAED,AAAA,iBAAiB,CAAC;EAChB,UAAU,EAAE,IAAI;CACjB;;AAED,AAAA,KAAK,CAAC;EACJ,MAAM,EAAE,KAAK;EACb,KAAK,EAAE,QAAQ;CAuBhB;;AAzBD,AAIE,KAJG,CAIH,EAAE,CAAC;EACD,KAAK,EAAE,OAAO;EACd,UAAU,EAAE,IAAI;EAChB,WAAW,EAAE,oBAAoB;EACjC,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;CAClB;;AAZH,AAcE,KAdG,CAcH,CAAC,CAAC;EACA,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,KAAK;EACb,UAAU,EAAE,IAAI;EAChB,WAAW,EAAE,MAAM;EACnB,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,MAAM;EACnB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,KAAK,EAAE,OAAO;CACf;;AAGH,AAAA,gBAAgB,CAAC;EACf,OAAO,EAAE,IAAI;EACb,KAAK,EAAE,IAAI;CAOZ;;AATD,AAIE,gBAJc,CAId,KAAK,CAAC;EACJ,YAAY,EAAE,IAAI;EAClB,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,IAAI;CACb",
    "sources": [
        "style.scss"
    ],
    "names": [],
    "file": "style.css"
}