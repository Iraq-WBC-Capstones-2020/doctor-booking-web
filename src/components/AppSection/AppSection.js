import React from 'react';
import { Row, Container } from 'react-bootstrap';
import appImg from './images/undraw_posting_photo_v65l.png';
import appStore from './images/AppStoreEng.png';
import googlePlay from './images/GooglePlayEng.png';
import './style.css';

function AppSection() {
  return (
    <div className="appContainer ">
      <Container className="applicationSection">
        <Row className="myRow">
          <div className="col-lg-6">
            <div className="text">
              <h2>Download our mobile apps (comming soon)</h2>
              <div className="underline"></div>
              <p>
                Our dedicated patient engagement app and web portal allow you to
                access information instantaneously (no tedeous form, long calls,
                or administrative hassle) and securely
              </p>
            </div>
            <div className="downloadButtons">
              <input alt="On App store" type="image" src={appStore} />
              <input alt="On Google play" type="image" src={googlePlay} />
            </div>
          </div>
          <div className="col-lg-6">
            <div className="applicationImage">
              <img alt="our app" className="img-fluid" src={appImg} />
            </div>
          </div>
        </Row>
      </Container>
    </div>
  );
}

export default AppSection;
