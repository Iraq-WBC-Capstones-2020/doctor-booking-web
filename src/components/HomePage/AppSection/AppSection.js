import React from 'react';
import { Row, Container } from 'react-bootstrap';
import appImg from './images/undraw_posting_photo_v65l.png';
import appStore from './images/AppStoreEng.png';
import googlePlay from './images/GooglePlayEng.png';
import '../HomePageGlobalStyle.scss';
import './style.css';

function AppSection() {
  return (
    <Container className="applicationSection">
      <Row className="myRow">
        <div className="col-xs-6">
          <div className="text">
            <h2>Download our Mobile Application (Comming soon)</h2>
            <hr className="grayLine w-100" />
            <p>
              Our dedicated patient engagement app and web portal allow you to
              access information instantaneously (no tedeous form, long calls,
              or administrative hassle) and securely
            </p>
          </div>
          <div className="downloadButtons">
            <input alt="On App store" type="image" src={appStore} />
            <input alt="On Google play" type="image" src={googlePlay} />
          </div>
        </div>
        <div className="col-xs-6">
          <div className="applicationImage">
            <img className="img-fluid" alt="our app" src={appImg} />
          </div>
        </div>
      </Row>
    </Container>
  );
}

export default AppSection;
