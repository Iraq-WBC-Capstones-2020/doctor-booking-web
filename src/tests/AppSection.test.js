import React from 'react';
import { render } from '@testing-library/react';
import AppSection from '../components/AppSection/AppSection';
import renderer from 'react-test-renderer';
it('render1', () => {
  render(<AppSection />);
});

it('renders correctly', () => {
  const tree = renderer.create(<AppSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
