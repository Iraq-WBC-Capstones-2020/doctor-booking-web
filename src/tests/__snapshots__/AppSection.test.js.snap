// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ApplicationSection container"
>
  <div
    className="myRow row"
  >
    <div
      className="col-xs-6"
    >
      <div
        className="text"
      >
        <h2>
          Download our mobile apps (comming soon)
        </h2>
        <p>
          Our dedicated patient engagement app and web portal allow you to access information instantaneously (no tedeous form, long calls, or administrative hassle) and securely
        </p>
      </div>
      <div
        className="download-buttons"
      >
        <input
          src="App Store - eng.png"
          type="image"
        />
        <input
          src="Google Play - eng.png"
          type="image"
        />
      </div>
    </div>
    <div
      className="col-xs-6"
    >
      <div
        className="Application-Image"
      >
        <img
          alt="our app"
          src="undraw_posting_photo_v65l.png"
        />
      </div>
    </div>
  </div>
</div>
`;
